extension/src/popup/helpers/useNetworkFees.ts

```diff
@@ -15,7 +15,7 @@ export const useNetworkFees = () => {
   const { networkUrl, networkPassphrase } = useSelector(
     settingsNetworkDetailsSelector,
   );
-  const [recommendedFee, setRecommendedFee] = useState("100");
+  const [recommendedFee, setRecommendedFee] = useState("");
```

Comment:
@aristidesstaffieri I noticed this defaulted to "100" from your PR here: https://github.com/stellar/freighter/pull/1376/files#diff-48e5eb06e07934afbdf9c6be1ad5d87bb1658ee0c72641914788d5655df34d2dR18

Was this for testing or was this supposed to fix a bug? I noticed that I was always getting my fee set to 100 XLM even though the fee stats from Horizon were coming back lower

Reply:
Yeah this was meant to fix a bug, I believe it was in the event that we couldn't get a fee then we would have no default but this value should be updated when the fees are fetched. 

After looking at this hook again, I think it's still flawed because there is no loading state so probably the initial value gets set and fee stats call never updates it due to the potential race condition if you confirm the screen before the fee resolves from the rpc.

Reply:
Oh gotcha. Cool - I'll add a fix in the next commit